libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -475,16 +475,16 @@ void AP_Baro::init(void)
     }
     
 #if HAL_WITH_UAVCAN
-    // If there is place left - allocate one UAVCAN based baro
-    if ((AP_BoardConfig::get_can_enable() != 0) && (hal.can_mgr != nullptr))
+    // TODO: If there is place left - allocate one UAVCAN based baro
+    /*if ((AP_BoardConfig::get_can_enable() != 0) && (hal.can_mgr != nullptr))
```

Comment:
These are preliminary as the whole structure changed and I commented out all code that was old